db/drivers/ogr/execute.c

```diff
@@ -109,7 +109,9 @@ int db__driver_execute_immediate(dbString * sql)
 	    }
 	    OGR_F_SetFieldString(hFeature, cols[i].index, value);
 	}
-	OGR_L_SetFeature(hLayer, hFeature);
+	if (0 != OGR_L_SetFeature(hLayer, hFeature))
```

Comment:
```suggestion
	if (OGR_L_SetFeature(hLayer, hFeature) != OGRERR_NONE)
```

Reply:
Good. I actually did consider using `OGRERR_NONE`,  but sided with using example from surrounding code.
Updated.

Reply:
I recommend following the OGR API for [OGR_L_SetFeature()](https://gdal.org/doxygen/ogr__api_8h.html#a5b380520d0999422987f06b6e31b9b00)